packages/frontend-2/components/viewer/controls/Left.vue

```diff
@@ -133,7 +133,7 @@
       :style="`width: ${widthClass};`"
     >
       <KeepAlive v-show="activePanel === 'models'">
-        <ViewerModelsPanel />
+        <ViewerModelsPanel v-model:sub-view="modelsSubView" />
```

Comment:
not necessarily within the scope of this task, but maybe we can also clean up some of those KeepAlives/v-shows we see in the viewer? the less inactive components we keep alive, the more performant the experience is